sal_interfaces/MTMount/MTMount_Commands.xml

```diff
@@ -3,201 +3,127 @@
-    <Version>3.0</Version>
-    <Author>Ander Ansuategi</Author>
     <EFDB_Topic>MTMount_command_moveToTarget</EFDB_Topic>
-    <Alias>moveToTarget</Alias>
-    <Device>target</Device>
-    <Property>position</Property>
-    <Action/>
-    <Value/>
-    <Explanation>http://sal.lsst.org/SAL/Commands/MTMount_command_moveToTarget.html</Explanation>
+    <Description>Go to a specified position and stop.</Description>
     <item>
-      <EFDB_Name>az_angle</EFDB_Name>
+      <EFDB_Name>azimuth</EFDB_Name>
       <Description>Azimuth angle</Description>
       <IDL_Type>double</IDL_Type>
       <Units>degree</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>el_angle</EFDB_Name>
+      <EFDB_Name>elevation</EFDB_Name>
       <Description>Elevation angle</Description>
       <IDL_Type>double</IDL_Type>
       <Units>degree</Units>
       <Count>1</Count>
     </item>
-    <item>
-      <EFDB_Name>cablewrap_orientation</EFDB_Name>
-      <Description>Cable Wrap orientation: Clockwise(CW), Counterclockwise(CCW)</Description>
-      <IDL_Type>string</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTMount</Subsystem>
-    <Version>3.0</Version>
-    <Author>Ander Ansuategi</Author>
-    <EFDB_Topic>MTMount_command_openMirrorCover</EFDB_Topic>
-    <Alias>openMirrorCover</Alias>
-    <Device>mirrorCover</Device>
-    <Property>position</Property>
-    <Action/>
-    <Value/>
-    <Explanation>http://sal.lsst.org/SAL/Commands/MTMount_command_openMirrorCover.html</Explanation>
-    <item>
-      <EFDB_Name>state</EFDB_Name>
-      <Description>Unused</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
+    <EFDB_Topic>MTMount_command_startTracking</EFDB_Topic>
+    <Description>Enable tracking. Once you issue this command you must call trackTarget at regular intervals until you are done tracking, at which point call stopTracking.</Description>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTMount</Subsystem>
-    <Version>3.0</Version>
-    <Author>Ander Ansuategi</Author>
     <EFDB_Topic>MTMount_command_trackTarget</EFDB_Topic>
-    <Alias>trackTarget</Alias>
-    <Device>target</Device>
-    <Property>position</Property>
-    <Action/>
-    <Value/>
-    <Explanation>http://sal.lsst.org/SAL/Commands/MTMount_command_trackTarget.html</Explanation>
+    <Description>Specify position, velocity and time (PVT) vector for all axes. This command is immediately reported as done.</Description>
     <item>
-      <EFDB_Name>az_angle</EFDB_Name>
-      <Description>Azimuth angle</Description>
+      <EFDB_Name>azimuth</EFDB_Name>
+      <Description>Mount azimuth angle at the specified time. The allowed range is the software limits (L1), approximately -270 to 270.</Description>
       <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
+      <Units>deg</Units>
```

Comment:
You replace `degree` here with `deg` but on lines 32 and 39 you keep using `degree`. Perhaps replace all occurrences of `degree` with `deg`?

Reply:
Good catch. I changed the two instances of degree to deg.